src/app/components/Rating/Rating.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import RatingInput from './RatingInput';
+import type { Story } from '@storybook/react';
+import type { RatingInputProps } from './RatingInput';
+
+export default {
+  title: 'Component/RatingInput',
+  component: RatingInput,
+
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+const Template: Story<RatingInputProps> = (args) => <RatingInput {...args} />;
+
+export const ConditionInput1 = Template.bind({});
```

Comment:
Not optimal naming maybe try:
```suggestion
export const ConditionRatingLow = Template.bind({});
```